<launch>
  <node name="down_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="down_cam" />
    <param name="camera_info_url" value="file://$(find env_percept)/config/down_cam.yaml" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="30" />
    <!-- <param name="autoexposure" value="false" /> --> 
    <param name="exposure" value="1" />
    <param name="brightness" value="1" />
  </node>
  <node pkg="image_proc" type="image_proc" name="down_image_proc" output="screen" ns="down_cam" />

  <!-- <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/down_cam/image_raw"/>
    <param name="autosize" value="true" /> 
  </node> -->
</launch>
